daemon/errors.go

```diff
@@ -342,6 +342,8 @@ func errToResponse(err error, snaps []string, fallback errorResponder, format st
 			snapName = err.Snap
 		case *snapstate.InsufficientSpaceError:
 			return InsufficientSpace(err)
+		case *snapstate.BusySnapError:
```

Comment:
probably warrants a unit test, you can extend TestErrToResponse in daemon/errors_test.go